shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -43,6 +45,19 @@ - (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
   return success;
 }
 
+- (void) createBkgReadableDir:(NSString*) path
```

Comment:
oh it sets property on directory and file... `[fm setAttributes:noProt ofItemAtPath:path error:nil];`

does it need to be on the directory? that seems wrong? also if the files are all in the same dir, it sets the property on it over and over?